test/Market.test.ts

```diff
@@ -1137,50 +1137,49 @@ describe('Market', () => {
 });
 describe('purchasing from a specified supplier', () => {
   it('should purchase supply from a specific supplier when they have enough supply', async () => {
-    const { bpNori, market, feePercentage, totalAmountOfSupply } =
-      await setupTest({
-        userFixtures: {
-          supplier: {
-            removalDataToList: {
-              removals: [
-                {
-                  amount: formatTokenAmount(10),
-                  supplierAddress: hre.namedAccounts.supplier,
-                }, // 2 removals each for 2 different suppliers
-                {
-                  amount: formatTokenAmount(10),
-                  supplierAddress: hre.namedAccounts.supplier,
-                },
-                {
-                  amount: formatTokenAmount(10),
-                  supplierAddress: hre.namedAccounts.investor1,
-                },
-                {
-                  amount: formatTokenAmount(10),
-                  supplierAddress: hre.namedAccounts.investor1,
-                },
-              ],
-            },
+    const { bpNori, market, totalAmountOfSupply } = await setupTest({
+      userFixtures: {
+        supplier: {
+          removalDataToList: {
+            removals: [
+              {
+                amount: formatTokenAmount(10),
```

Comment:
you can also delete this one I believe (see [here](https://github.com/nori-dot-eco/contracts/blob/7046e8446201ffb20664b74dc152ea709308a17f/test/checkout.int.t.sol#L409))